client/a8c-for-agencies/sections/marketplace/common/pressable-usage-limit-notice/index.tsx

```diff
@@ -49,13 +51,21 @@ export const PressableUsageLimitNotice = () => {
 			className="pressable-usage-limit-notice"
 		>
 			<div>
-				{ translate(
-					'Your Pressable plan has exceeded its allocated %s limit. Consider upgrading your plan to avoid additional fees.',
-					{
-						args: [ limitTypeHumanised ],
-						comment: 'The limit type is storage or traffic which is already translated',
-					}
-				) }
+				{ isLimitExceeded
+					? translate(
+							'Your Pressable plan has exceeded its allocated %s limit. Consider upgrading your plan to avoid additional fees.',
```

Comment:
Nitpicking: It is probably better to use `%(usageType)s` instead of `%s`, as it is much easier for the translator to read with context.

Reply:
I have updated it now: https://github.com/Automattic/wp-calypso/pull/95211/commits/ab35eb5d0c9012c13ec252602ec1a0bda73a8f79